mscore/timeline.cpp

```diff
@@ -900,7 +915,7 @@ void Timeline::drawGrid(int global_rows, int global_cols)
                         part_name = part_list.at(row)->instrumentName();
 
                   graphics_rect_item->setToolTip(initial_letter + QString(" ") + QString::number(curr_measure->no() + 1) + QString(", ") + part_name);
-                  graphics_rect_item->setPen(QPen(QColor(Qt::lightGray)));
+                  graphics_rect_item->setPen(QPen(preferences.isThemeDark() ? mainColorDark : mainColorLight));
```

Comment:
It is better to make a method for returning a color (for example `mainColor()`) and in it do a check and return the desired color, instead of duplicating the logic